src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/CrossTarget/CrossTargetRuleSubscriberBase.cs

```diff
@@ -243,10 +253,18 @@ private IEnumerable<string> GetWatchedRules(RuleHandlerType handlerType)
                                                   isActiveContext,
                                                   ruleChangeContext)
                                      .ConfigureAwait(true);
+
+                        // check full set of handler rules for changes
+                        _treeTelemetryService.ObserveHandlerRulesChanges(
+                            projectContextToUpdate.TargetFramework,
+                            handler.GetRuleNames(RuleHandlerType.DesignTimeBuild), 
```

Comment:
>DesignTimeBuild [](start = 65, length = 15)

here you also can have evaluation rules not only DesignTime. Did you want to use a new local variable that you created : "handlerRules" . It would contain current rules to be handled. Otherwise you would report telemetry for evaluation not for DT build.